packages/ui/src/components/editor/properties/render/index.tsx

```diff
@@ -123,6 +127,23 @@ export const RenderSettingsEditor: EditorComponentType = (props) => {
     })
   )
 
+  useEffect(() => {
```

Comment:
Unfortunately having this here means we will need to open the render settings propreties UI in order for the light to be updated, which kind of defeats the purpose. We can just move this to a reactor in a system somewhere to ensure it's always running

Reply:
If the value points to a light that doesn't exist it shouldn't matter until that UI is opened anyways.

Reply:
that's a fair point!